frontend-zs/src/components/stories/StoryList.component.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import { Button, Container, Row, Col, Card, Media, Image, Form } from 'react-bootstrap'
+import { Link } from 'react-router-dom'
+import likeIcon from '../../icon_like.png'
+import './Story.scss'
+
+const StoryCard = props => (
+    <Col sm={6}>
+        <Card className="mb-4">
+            <Card.Header style={{ backgroundColor: '#5F696A' }} />
+            <Card.Body>
+                <Card.Title><strong>{props.story.title}</strong></Card.Title>
+                <Card.Subtitle className="mb-2 text-muted">
+                    <strong>{props.story.author}</strong>
+                </Card.Subtitle>
+                <Card.Text>{props.story.shortDescription}</Card.Text>
+                <Container>
+                    <Row className="justify-content-between">
+                        <Col>
+                            <Button variant="flat" size="sm" active>
+                                <Link to={"/geschichte/" + props.story._id}>
+                                    Geschichte lesen
+                                </Link>
+                            </Button>
+                        </Col>
+                        <Media>
+                            <Image
+                                className="mr-1"
+                                src={likeIcon}
+                                width="23"
+                                height="23"
+                                fluid
+                            />
+                            <Media.Body>
+                                <h5><strong>{props.story.numberOfLikes.length}</strong></h5>
+                            </Media.Body>
+                        </Media>
+                    </Row>
+                </Container>
+            </Card.Body>
+        </Card>
+    </Col>
+)
+
+export default class StoryList extends React.Component {
+    constructor(props) {
+        super(props)
+
+        this.onChangeSearchCriteria = this.onChangeSearchCriteria.bind(this)
+
+        /* TODO: exchange hard coded data with data from the database */
```

Comment:
Since the endpoint is already merged to develop, I'd suggest extracting the data from the db in this PR.